xml/decoderIndex.xml

```diff
@@ -1,184 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet href="/xml/XSLT/DecoderID.xsl" type="text/xsl"?>
 <decoderIndex-config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
-  <!--Written by JMRI version 5.5.5plus+jake+20231015T1935Z+R432d7126c8 on Sun Oct 15 15:35:13 EDT 2023-->
-  <decoderIndex version="59">
+  <!--Written by JMRI version 5.5.5plus+jake+20231016T1819Z+R27d2317b3e on Mon Oct 16 14:19:53 EDT 2023-->
+  <decoderIndex version="1">
     <!--The manufacturer list is from the nmra_mfg_list.xml file-->
-    <mfgList nmraListDate="2021-03-06" updated="2023-01-25" lastadd="169">
+    <mfgList>
       <manufacturer mfg="NMRA" mfgID="999" />
-      <manufacturer mfg="A-Train Electronics" mfgID="137" />
-      <manufacturer mfg="AE Electronic Ltd" mfgID="169" />
-      <manufacturer mfg="AMW" mfgID="19" />
```

Comment:
This look strange to me. Almost all of the manufacturers, except NMRA, are removed by this PR. Is this correct?

Reply:
You're right, that's odd.  Looking into it....